apps/src/templates/amazonFutureEngineerEligibility/amazonFutureEngineerEligibilityForm.jsx

```diff
@@ -9,6 +9,7 @@ import FieldGroup from '../../code-studio/pd/form_components/FieldGroup';
 import SingleCheckbox from '../../code-studio/pd/form_components/SingleCheckbox';
 import color from '@cdo/apps/util/color';
 import {isEmail} from '@cdo/apps/util/formatValidation';
+import {STATES} from '@cdo/apps/geographyConstants';
```

Comment:
This constant contains only full state names.  The data formats we agreed on with AFE use two-letter state codes. If we're going to use the full state name here on the client, let's use [`get_us_state_abbr_from_name`](https://github.com/code-dot-org/code-dot-org/blob/a36c1ddcb536750558c640330e13395555eb1924/lib/state_abbr.rb#L21) in our submit controller action to convert it before sending it out.

It would be a good cleanup to deduplicate geographyConstants.js with the state list we have in Ruby.

Reply:
Sorry, did not realize we were supposed to be sending state codes to them in the first place. Should be fixed now!